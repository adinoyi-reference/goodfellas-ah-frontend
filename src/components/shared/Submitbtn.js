import React from 'react';

const Submitbtn = () => (
  <div>
    <input type="submit" className="btn btn-block mt-4 resetbtn" />
  </div>
);

export default Submitbtn;
