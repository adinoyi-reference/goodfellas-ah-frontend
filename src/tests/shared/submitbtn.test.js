
import React from 'react';
import { mount } from 'enzyme';
import toJson from 'enzyme-to-json';
import Submitbtn from '../../components/shared/Submitbtn';
import Root from '../../root';

let wrapped;

beforeEach(() => {
  wrapped = mount(
    <Root>
      <Submitbtn />
    </Root>,
  );
});

afterEach(() => wrapped.unmount());
describe('Submit btn', () => {
  describe('render', () => {
    test('container should render as expected', () => {
      const tree = toJson(wrapped);
      expect(tree).toMatchSnapshot();
    });
  });
});
