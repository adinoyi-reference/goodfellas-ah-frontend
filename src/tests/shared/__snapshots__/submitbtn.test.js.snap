// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Submit btn render container should render as expected 1`] = `
<_default>
  <Provider
    store={
      Object {
        "dispatch": [Function],
        "getState": [Function],
        "replaceReducer": [Function],
        "subscribe": [Function],
        Symbol(observable): [Function],
      }
    }
  >
    <Submitbtn>
      <div>
        <input
          className="btn btn-block mt-4 resetbtn"
          type="submit"
        />
      </div>
    </Submitbtn>
  </Provider>
</_default>
`;
